templates/pages/paper.jsx

```diff
@@ -42,6 +42,7 @@ var Issue = React.createClass({
         if (response.status >= 400) {
             throw new Error("Bad response from server");
         }
+        alert("Form Submitted Successfully!");
```

Comment:
Hi @jinankjain! I'd rather not use an alert here. I was hoping for an update in the interface.